assets/report_template.Rmd

```diff
@@ -1307,15 +1308,19 @@ Folder [qiime2/alpha-rarefaction](../qiime2/alpha-rarefaction) contains the data
 
 <!-- Subsection on diversity analysis -->
 
-```{r, eval = !isFALSE(params$diversity_indices_beta), results='asis'}
+```{r, eval = !isFALSE(params$diversity_indices_alpha) || !isFALSE(params$diversity_indices_beta), results='asis'}
 diversity_indices_depth <- readLines(params$diversity_indices_depth)
 
 cat(paste0("
 ## Diversity analysis
 
 Diversity measures summarize important sample features (alpha diversity) or differences between samples (beta diversity).
 Diversity calculations are based on sub-sampled data rarefied to ",diversity_indices_depth, " counts.
```

Comment:
Should `",diversity_indices_depth, "` actually be an R inline, i.e. ``r diversity_indices_depth`` (there should be backticks around that; couldn't format it correctly)?

Reply:
That would be true if that lines weren't in the R block that starts in line 1310/1311. And I did put it into that R block because that complete text is conditional (probably there better ways to achieve the same?).